src/ctia/entity/asset_mapping.clj

```diff
@@ -97,29 +105,67 @@
    :valid_time.start_time
    :valid_time.end_time])
 
-(def asset-mapping-routes
-  (entity-crud-routes
-   {:entity                   :asset-mapping
-    :new-schema               NewAssetMapping
-    :entity-schema            AssetMapping
-    :get-schema               PartialAssetMapping
-    :get-params               AssetMappingGetParams
-    :list-schema              PartialAssetMappingList
-    :search-schema            PartialAssetMappingList
-    :external-id-q-params     AssetMappingByExternalIdQueryParams
-    :search-q-params          AssetMappingSearchParams
-    :new-spec                 :new-asset-mapping/map
-    :realize-fn               realize-asset-mapping
-    :get-capabilities         :read-asset-mapping
-    :post-capabilities        :create-asset-mapping
-    :put-capabilities         :create-asset-mapping
-    :delete-capabilities      :delete-asset-mapping
-    :search-capabilities      :search-asset-mapping
-    :external-id-capabilities :read-asset-mapping
-    :can-aggregate?           true
-    :histogram-fields         asset-mapping-histogram-fields
-    :enumerable-fields        asset-mapping-enumerable-fields
-    }))
+(s/defn additional-routes [{{:keys [get-in-config]}          :ConfigService
+                            {:keys [read-store write-store]} :StoreService
+                            :as                              services} :- APIHandlerServices]
+  (routes
+   (PATCH "/expire/:id" []
```

Comment:
IMHO using the `PATCH` verb here is misleading, in the REST world, `PATCH` is generally used by an API customer to supply a partial update for an entity, typically for such operations which are outside of the REST scope it is preferable to use `POST`.